nl/guides/teachers/ufora/README.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Dodona en Ufora
+description: "Tutorial: Dodona en Ufora"
+---
+
+# Dodona en Ufora
+
+::: warning UGent-only
+Deze pagina gaat over het koppelen van Dodona met Ufora, de leeromgeving van de Universiteit Gent.
+:::
+
+Sinds 2020-2021 is het mogelijk om Dodona oefeningen en reeksen in te voegen in de inhoudstool van Ufora. Studenten die doorklikken worden automatisch ingelogd met hun UGent account. In een latere fase voorzien we integratie met de scoremodule van Ufora zodat we het resultaat van de ingediende oplossingen kunnen terugsturen naar Ufora.
+
+## 1. Open je cursus op Ufora
+
+Surf naar [ufora.ugent.be](https://ufora.ugent.be) en navigeer naar je cursus. Open vervolgens  de inhoudstool en kies de gewenste module in de navigatiebalk.
+
+## 2. Open de oefeningenkiezer
+
+Klik op het menu `Bestaande activiteiten` en vervolgens op `Dodona`. Er verschijnt nu een dialoogvenster om oefeningen te selecteren.
+
+::: tip Geen Dodona?
+De Dodonamodule is niet in alle faculteiten standaard ingeschakeld. Zie je geen Dodona-optie in het menu? Contacteer ons dan via dodona@ugent.be met vermelding van je cursuscode.
```

Comment:
De Dodona-module is slechts ingeschakeld voor enkele faculteiten.

Reply:
```suggestion
De Dodona-module is niet in alle faculteiten standaard ingeschakeld. Zie je geen Dodona-optie in het menu? Contacteer ons dan via dodona@ugent.be met vermelding van je cursuscode.
```